Description: Add missing sys/types.h include to storage.h
 Without this missing include, ofono would fail to build from source under
 some versions of gcc (and/or build flags).
Author: Stéphane Graber <stgraber@ubuntu.com>

Origin: vendor
Forwarded: no
Reviewed-By: Stéphane Graber <stgraber@ubuntu.com>
Last-Update: 2013-01-28

--- ofono-1.12.orig/src/storage.h
+++ ofono-1.12/src/storage.h
@@ -25,6 +25,7 @@
 #define TFR
 #endif
 
+#include <sys/types.h>
 #include <fcntl.h>
 
 int create_dirs(const char *filename, const mode_t mode);
